apps/server/src/modules/contributions/index.ts

```diff
@@ -0,0 +1,64 @@
+import { ContributionsService } from '../contribs';
+import { contributionsDBService } from '../../common/db';
+
+class ContributionManagerImpl implements ContributionsService {
```

Comment:
we probably don't need the `ContributionService` anymore now that we don't have many Contribution Managers, we could just use `ContributionManager` directly and make it request the DB in its constructor using a token